### DISCUSSION TITLE
popup edit on double click

### DISCUSSION THREAD

**Discussion by r35p3ct** [quasarframework/quasar#16650] [General - Components / Directives / etc]
There is a popupedit in the q-table that appears when you click on a cell, how can I change it so that it show on dclick? I want to use single click for my logic.

```
<template v-slot:body-cell="props">
          <q-td :props="props" :class="colors(props)">
            {{ props.value }}

            <q-popup-edit ref="popupEdit" v-if="!isNaN(props.col.field)" v-model="props.row[props.col.name]" v-slot="scope">
              <q-input v-model="scope.value" dense autofocus clearable @keyup.enter="scope.set" />

            </q-popup-edit>
          </q-td>
</template>
```

**Comment by metalsadman** [comment]
@r35p3ct https://codepen.io/metalsadman/pen/KKJGbba?editors=1010

**Reply by r35p3ct** [reply]
@metalsadman Thanks for the hint, but I still can’t get a working version with a table, the cell value is not transferred to the editor, and the clearable property disappears altogether.
https://codepen.io/Mr-Deito/pen/LYqgaVG?editors=1011

`<template v-slot:body-cell="props">
 <q-td :props="props" @click="onClick">
            {{ props.row[props.col.name] }}
            <q-popup-edit 
                          v-model="props.row[props.col.name]"
                          v-slot="scope">
              <q-input v-model="scope.value" dense autofocus clearable @keyup.enter="scope.set" />
            </q-popup-edit>
          </q-td>
</template>`
